### DISCUSSION TITLE
How to integrate Source Guardian?

### DISCUSSION THREAD

**Discussion by tuannhsmartosc** [markshust/docker-magento#767] [General]
Hi @markshust,
I want to install Source Guardian for PHP 8.1 in this docker, but I have never done it before. 
Can you please help set it up in your docker-compose.yml file?

Thank you in advance!

**Comment by rangerz** [comment]
Try `bin/install-php-extensions sourceguardian` from https://github.com/markshust/docker-magento/commit/1786c585b96044c95b39c2579ad5e9cedb370474
